cypress/e2e/internal/billing/supplementary/reissue-sroc.cy.js

```diff
@@ -0,0 +1,268 @@
+
+    // Reissue the bill
+    cy.get('p > .govuk-button').click()
+
+    // What date do you need to reissue a bill from?
+    // we use the current date to demonstrate the date is based on when the bill was created, not the billing period
+    // the bill is for then click continue
+    const currentDate = new Date()
+    cy.get('input#fromDate-day').type(currentDate.getDate())
+    cy.get('input#fromDate-month').type(currentDate.getMonth() + 1)
+    cy.get('input#fromDate-year').type(currentDate.getFullYear())
+    cy.get('.govuk-button').contains('Continue').click()
+
+    // There are 2 bills available for reissue to Big Farm Co Ltd 02
+    // confirm both bills are returned then click the change link
+    cy.get('#main-content > div > div > h1').should('contain.text', 'There are 2 bills available for reissue')
+    cy.get('div:nth-child(2) > dd.govuk-summary-list__actions > a').contains('Change').click()
+
+    // Select the bills you need to reissue
+    // for the purposes of this test we only need to reissue one bill. Any more and we are just slowing down the
+    // test. Untick the second one and continue
+    cy.get('input#selectedBillIds-2').uncheck()
+    cy.get('.govuk-button').contains('Continue').click()
+
+    // There is 1 bill available for reissue to Big Farm Co Ltd 02
+    // confirm just 1 bill is now selected then click confirm
+    cy.get('#main-content > div > div > h1').should('contain.text', 'There is 1 bill available for reissue')
+    cy.get('.govuk-button').contains('Confirm').click()
+
+    // You’ve marked 1 bill for reissue
+    // confirmation that we have selected 1 bill for reissue. Click create a supplementary bill run
+    cy.get('#main-content > div > div > div > h1').should('contain.text', 'You’ve marked 1 bill for reissue')
+    cy.get('#main-content > div > div > a').contains('Create a supplementary bill run').click()
+
+    // -------------------------------------------------------------------------
+    cy.log('Create, confirm and send the SROC supplementary bill run that will reissue the bill')
+
+    // Which kind of bill run do you want to create?
+    // choose Supplementary and continue
+    cy.get('input#selectedBillingType-2').click()
+    cy.get('form > .govuk-button').contains('Continue').click()
+
+    // Select the region
+    // choose Test Region and continue
+    cy.get('input#selectedBillingRegion-9').click()
+    cy.get('form > .govuk-button').contains('Continue').click()
+
+    // "There is already a bill run in progress for this region" -- which is the old scheme bill run
+    // click the Bill runs menu link
+    cy.get('#navbar-bill-runs').contains('Bill runs').click()
+
+    // Bill runs
+    // The sroc bill run we created should be the top result. Once it has finished building its status will be `Ready`
+    // so we reload until the text is present.
+    cy.reloadUntilTextFound('tr:nth-child(1) > td:nth-child(6) > strong', 'Ready', 10)
```

Comment:
When I run this test locally it's failing for me on line 206 as it's creating an `Empty` bill run.

Reply:
I found this issue was caused by the following environment variable not being in my .env in my system repo: `ENABLE_REISSUING_BILLING_BATCHES=true`. The test passed with that added.